gc-policy.yaml

```diff
@@ -1,15 +1,31 @@
+# Production
+testing:
+    cloud-uploads: 2y
+    containers: 2m
+stable:
+    cloud-uploads: 2y
+    containers: 2m
+next:
+    cloud-uploads: 2y
+    containers: 2m
+# Development
 testing-devel:
     cloud-uploads: 2y
     containers: 2w
 next-devel:
     cloud-uploads: 2y
     containers: 2w
-branched:
+# Mechanical
+rawhide:
     cloud-uploads: 1y
     containers: 2w
-rawhide:
+    images: 3y
+    build: 4y
+branched:
     cloud-uploads: 1y
     containers: 2w
+    images: 1y
+    build: 1y
```

Comment:
if these are the same value it doesn't mean anything does it?

Reply:
Nope technically, we could just delete `images` here since the `build` is gonna delete everything anyways.